tgui/packages/tgui/interfaces/Orbit.js

```diff
@@ -0,0 +1,181 @@
+import { Button, Input, Section } from '../components';
+import { Window } from '../layouts';
+import { useBackend, useLocalState } from '../backend';
+
+const PATTERN_DESCRIPTOR = / \[(?:ghost|dead)\]$/;
+const PATTERN_NUMBER = / \(([0-9]+)\)$/;
+
+const searchFor = bySearch => thing => {
+  return thing.name.toLowerCase().startsWith(bySearch.toLowerCase());
```

Comment:
I don't know shit about JS tbh, but couldn't this be a regex search? That way you don't need to exactly type the starting characters of your search.

Reply:
It could just as easily just check contains, I can change that late.r